### ISSUE TITLE
[Suggestion] simulating radiation.

### ISSUE THREAD

**Issue by Romanmir** [erogenousbeef/BigReactors#511]
My suggestion involves simulating radiation. It stems from the weirdness that comes from standing in an almost fully formed reactor with full fuel rods, and taking no damage. Maybe it could be a config option.

If the control rods have any fuel in them while the multi-block is not complete, then damage the player in proportion to how full the fuel rods are if the player is within one block of a fuel rod, multiplied by the efficiency rating for the reactor. (eg. assuming 100% full fuel rods, at 100% efficiency, the damage radius is one block. at 496% efficiency, the damage radius is almost 5 blocks distant. conversely, if the fuel rods are only 25% full, then multiply the above 5 block distance by 25%.)

You could also consider reducing the amount of radiation taken based on the distance to the nearest fuel rod.

Then when the multiblock forms, remove the radiation damage altogether.

I'd imagine that one heart per 10 ticks might be a good place to start with how much damage is taken when the fuel rods are full. You could also apply negative effects as well (wither/slowed/blind, whatever.)

**Comment by Zedrimar** [comment]
I honestly think that this is a really good idea!!!! :)

also I would like to add that damage when carrying the fuel without protection should also cause radiation poisening

**Comment by RogueLogix** [comment]
very good idea. i think radiation protection gear would be amazing as well so it does not affect you. made of like graphite and glass or something.

**Comment by Zedrimar** [comment]
@killerall5385 Lead  and glass is a better idea as in reality it actually stops radiation unlike carbon based things like graphite (another form of carbon).

**Comment by RogueLogix** [comment]
true but i don't think @erogenousbeef wants to add lead to the mod. not to say that that recipe could not be added and the other removed if there is lead already.

**Comment by Zedrimar** [comment]
just saying, lead keeps it semi realistic because it is used more often then other materials. If lead is used, it can use existing ingots from other mods that have it or use this mod's ingots if no mod adds it and or adds it to the ore dict. Graphite has little resistance to the Gamma radiation thus alot more has to be used, making the player far bulkier and slower. It would be better to use iron armor and make a thicker version so it can stop gamma rays. As for the fuel, lets assume that it is a low alpha emitter like uranium 238, so a recipe for gloves (made out of wool) will need to be added (alpha particles are easily blocked by clothing). If the emitted particles are beta, metal or graphite "gloves" will need to be added.

**Comment by RogueLogix** [comment]
did i not say that is could rely on other mods for lead." if there is lead already" oh yeah i did. i don't know to much about how radiation works i just said it would be a good idea to add some way to protect yourself from it.

**Comment by Zedrimar** [comment]
yeah I understand, and sorry about earlier I didn't pay attention to the fact that you said that already
